src/inc/pedecoder.h

```diff
@@ -338,6 +339,8 @@ class PEDecoder
     TADDR GetVirtualSectionsTable(COUNT_T *pSize = NULL) const;
 #endif // FEATURE_PREJIT
 
+    CORCOMPILE_DEPENDENCY* GetNativeDependencies(COUNT_T* pCount = NULL) const;
```

Comment:
Does everything in the `CORCOMPILE_DEPENDENCY` structure make sense for R2R?

Also, we have all structures like this that are part of R2R format defined in `src\inc\readytorun.h`.

Reply:
Made a separate data structure for R2R.